umcontroller/umcontroller_internal_test.go

```diff
@@ -183,13 +194,25 @@ func TestNormalUpdateWithReboot(t *testing.T) {
 		case <-stopCh:
 			if stream.step == StepRebootOnUpdate {
 				handler, stopCh, err = newUmHandler("testUM2", &stream, eventChannel, pb.UmState_UPDATED)
+				if err != nil {
+					t.Errorf("Can't create um handler: %s", err)
```

Comment:
Could you use `UM` instead of `um` in logs in all cases?  Fix it in other cases as well.

Reply:
changed